README.md

```diff
@@ -132,20 +137,22 @@ The URLs to download the data are constructed according to the following pattern
 
 *For benchmark datasets (reference output)*, the pattern will be similar to https://storage.ecmwf.europeanweather.cloud/s2s-ai-challenge/data/training-output-benchmark/{parameter}-weeks-{weeks}.nc (NOT AVAILABLE YET)
 
-- {datasetname} : In the URLs the dataset name must follow the ML naming (training-input" or "training-output-reference" or "training-output-benchmark").
-- {format} is "netcdf". Training output is also available as GRIB file,  using format=grib and replacing ".nc" by ".grib"
-- {parameter} is "t2m" for surface temperature at 2m, "tp" for total precipitation using CF convention.
-- {origin} : "ecmwf" or "eccc" or "ncep".
-- {fctype} : "hindcast" ("forecast" for test dataset).
-- {weeks} is "34" or "56".
-- YYYYMMDD is the date of main forecast time in the file.
+
+- {datasetname} : In the URLs the dataset name must follow the ML naming (`training-input` or `training-output-reference` or `training-output-benchmark`).
+- {format} is `netcdf`. Training output is also available as GRIB file,  using `format='grib'` and replacing `".nc"` by `".grib"`
+- {parameter} is `t2m` for surface temperature at 2m, "tp" for total precipitation using CF convention.
+- {origin} : `ecmwf` or `eccc` or `ncep` <!-- we should have a clean table for this once -->
+- {fctype} : "hindcast" ("forecast" for test dataset). <!-- is this needed in the API? IMO not -->
```

Comment:
can we delete `fctype`? this is not needed anymore, at least in the external API? @floriankrb